src/components/ReactGoogleMaps.css

```diff
@@ -28,7 +28,6 @@
 
 .search-bar-container {
   position: relative;
-  pointer-events: none;
```

Comment:
There will be a merge conflict because ReactGoogleMaps.css was converted to a module. This line to remove pointer-events is the only relevant change that need to be made.